cap/modules/deposit/api.py

```diff
@@ -96,40 +102,40 @@
 )
 from .review import Reviewable
 
-_datastore = LocalProxy(lambda: current_app.extensions['security'].datastore)
+_datastore = LocalProxy(lambda: current_app.extensions["security"].datastore)
 
 PRESERVE_FIELDS = (
-    '_deposit',
-    '_buckets',
-    '_files',
-    '_review',
-    '_experiment',
-    '_access',
-    '_user_edited',
-    '_fetched_from',
-    'control_number',
-    'general_title',
-    '$schema',
+    "_deposit",
```

Comment:
Please let me know the reason for replacement for `"` here instead `'`

Reply:
its the formatter added no?

Reply:
We have it configured for single quote.